projects/extension/src/background/index.ts

```diff
@@ -12,18 +12,60 @@ import {
   ToExtension,
 } from "@substrate/connect-extension-protocol"
 
-// Note that this list doesn't necessarily always have to match the list of well-known
-// chains in `@substrate/connect`. The list of well-known chains is not part of the stability
-// guarantees of the connect <-> extension protocol and is thus allowed to change
-// between versions of the extension. For this reason, we don't use the `WellKnownChain`
-// enum from `@substrate/connect` but instead manually make the list in that enum match
-// the list present here.
-export const wellKnownChains: Map<string, string> = new Map<string, string>([
-  [polkadot.id, JSON.stringify(polkadot)],
-  [ksmcc3.id, JSON.stringify(ksmcc3)],
-  [rococo_v2_2.id, JSON.stringify(rococo_v2_2)],
-  [westend2.id, JSON.stringify(westend2)],
-])
+const setupStorageBootnodes = (): Map<string, string> => {
+  const polkadot_cp = Object.assign({}, polkadot)
+  const ksmcc3_cp = Object.assign({}, ksmcc3)
+  const westend2_cp = Object.assign({}, westend2)
+  const rococo_cp = Object.assign({}, rococo_v2_2)
+
+  chrome.storage.local.get(
+    [
+      "bootNode_".concat(polkadot_cp.id),
+      "bootNode_".concat(ksmcc3_cp.id),
+      "bootNode_".concat(westend2_cp.id),
+      "bootNode_".concat(rococo_cp.id),
+    ],
+    (result) => {
+      let i = "bootNode_".concat(polkadot_cp.id)
+      if (result[i]) {
+        polkadot_cp.bootNodes = result[i]
+      } else {
+        chrome.storage.local.set({ [i]: polkadot_cp.bootNodes })
```

Comment:
```suggestion
      } else {
        chrome.storage.local.set({ [i]: polkadot_cp.bootNodes })
```

Well, we don't need to do this, do we?

Reply:
🤦🏽 Thank you... seriously sometimes I get surprised on these kind of misses from me 🤦🏽